sentry/backup.json

```diff
@@ -0,0 +1 @@
userip","pk":1,"fields":{"user":1,"ip_address":"192.168.65.1","country_code":null,"region_code":null,"first_seen":"2024-07-24T17:50:09.894Z","last_seen":"2024-07-25T15:54:04.906Z"}},{"model":"sentry.userrole","pk":1,"fields":{"date_updated":"2024-07-24T17:21:46.345Z","date_added":"2024-07-24T17:21:46.345Z","name":"Super Admin","permissions":"['broadcasts.admin', 'users.admin', 'options.admin']"}},{"model":"sentry.userroleuser","pk":1,"fields":{"date_updated":"2024-07-24T17:23:25.251Z","date_added":"2024-07-24T17:23:25.251Z","user":1,"role":1}},{"model":"sentry.projectteam","pk":1,"fields":{"project":1,"team":1}},{"model":"sentry.projectteam","pk":2,"fields":{"project":2,"team":1}},{"model":"sentry.organizationmember","pk":1,"fields":{"organization":1,"user_id":1,"email":null,"role":"owner","flags":"0","token":null,"date_added":"2024-07-24T17:23:25.030Z","token_expires_at":null,"has_global_access":true,"inviter_id":null,"invite_status":0,"type":50,"user_is_active":true,"user_email":"admin@tanf.com"}},{"model":"sentry.environment","pk":1,"fields":{"organization_id":1,"name":"production","date_added":"2024-07-25T13:58:52.695Z"}},{"model":"sentry.environment","pk":2,"fields":{"organization_id":1,"name":"prod","date_added":"2024-07-25T15:56:14.474Z"}},{"model":"sentry.email","pk":1,"fields":{"email":"admin@tanf.com","date_added":"2024-07-24T17:23:25.004Z"}},{"model":"sentry.counter","pk":1,"fields":{"project":2,"value":1}},{"model":"sentry.actor","pk":1,"fields":{"type":0,"user_id":null,"team":1}},{"model":"sentry.useremail","pk":1,"fields":{"user":1,"email":"admin@tanf.com","validation_hash":"b0eDHicDKe4xGwYmtl3rOokz29WTv0qL","date_hash_added":"2024-07-24T17:23:25.002Z","is_verified":false}},{"model":"sentry.rule","pk":1,"fields":{"project":1,"environment_id":null,"label":"Send a notification for new issues","data":"{\"match\":\"all\",\"conditions\":[{\"id\":\"sentry.rules.conditions.first_seen_event.FirstSeenEventCondition\"}],\"actions\":[{\"id\":\"sentry.mail.actions.NotifyEmailAction\",\"targetType\":\"IssueOwners\",\"targetIdentifier\":null,\"fallthroughType\":\"ActiveMembers\"}]}","status":0,"source":0,"owner":null,"date_added":"2024-07-24T17:21:46.406Z"}},{"model":"sentry.rule","pk":2,"fields":{"project":2,"environment_id":null,"label":"Send a notification for new issues","data":"{\"match\":\"all\",\"conditions\":[{\"id\":\"sentry.rules.conditions.first_seen_event.FirstSeenEventCondition\"}],\"actions\":[{\"id\":\"sentry.mail.actions.NotifyEmailAction\",\"targetType\":\"IssueOwners\",\"targetIdentifier\":null,\"fallthroughType\":\"ActiveMembers\"}]}","status":0,"source":0,"owner":null,"date_added":"2024-07-25T13:54:44.108Z"}},{"model":"sentry.organizationmemberteam","pk":1,"fields":{"team":1,"organizationmember":1,"is_active":true,"role":null}},{"model":"sentry.environmentproject","pk":1,"fields":{"project":2,"environment":1,"is_hidden":null}},{"model":"sentry.environmentproject","pk":2,"fields":{"project":2,"environment":2,"is_hidden":null}}]
```

Comment:
This works as a settings file for local in conjunction with settings in local.py